schema.sql

```diff
@@ -1,5 +1,12 @@
-/* Database schema to keep the structure of entire database. */
+--  Create a database named vet_clinic
+CREATE DATABASE vet_clinic;
 
+-- Create a table named animals with the following columns:
 CREATE TABLE animals (
-    name varchar(100)
+    id INTEGER,
```

Comment:
- [ ] [OPTIONAL] Please mark the id column as the primary key to identify each record uniquely.
```
PRIMARY KEY (id)
```